package.json

```diff
@@ -92,11 +92,11 @@
     "packages/perf-benchmarks"
   ],
   "engines": {
-    "node": ">=10.13.0 <11.0.0",
-    "yarn": ">=1.10.1"
+    "node": ">=10.16.3 <10.17.0",
```

Comment:
Why do we restrict the version of `node` and `yarn` instead of relying on volta for that?